c/extras/py_ftrl.cc

```diff
@@ -23,127 +23,220 @@
 #include "frame/py_frame.h"
 #include "python/float.h"
 #include "python/int.h"
+#include "python/tuple.h"
 
 namespace py {
 
-PKArgs Ftrl::Type::args___init__(0, 0, 9, false, false,
-                                 {"a", "b", "l1", "l2", "d", "n_epochs",
-                                 "inter", "hash_type", "seed"},
-                                 "__init__", nullptr);
-
-void Ftrl::m__init__(PKArgs& args) {
-  FtrlModelParams fmp = FtrlModel::fmp_default;
-
-  if (!(args[0].is_undefined() || args[0].is_none())) {
-    fmp.a = args[0].to_double();
-  }
+PKArgs PyFtrl::Type::args___init__(0, 0, 10, false, false,
+                                   {"params", "alpha", "beta", "lambda1",
+                                   "lambda2", "d", "n_epochs", "inter",
+                                   "hash_type", "seed"},
+                                   "__init__", nullptr);
```

Comment:
You should allow `params` to be an unnamed argument too (or perhaps, only unnamed?) `core.Ftrl(model_params)`

Reply:
Yes, it will look nicer than. Thanks for suggestion.

Reply:
Fixed